draft-private-access-tokens.md

```diff
@@ -538,26 +636,17 @@ an Origin that is served by the Issuer
 If any of these conditions is not met, the Issuer MUST return an HTTP 400 error to the Mediator,
 which will forward the error to the client.
 
-If the conditions are met, the Issuer then applies its policy for the Client request, by checking
-for state associated with the ANON_CLIENT_ID. If there is previous state associated with the
-ANON_CLIENT_ID, the Issuer first validates two conditions:
-
-- The decrypted ORIGIN_NAME for this request matches the ORIGIN_NAME stored from previous requests
-using this ANON_CLIENT_ID.
-- The decrypted ANON_ORIGIN_ID_PRIME for this request matches the ANON_ORIGIN_ID_PRIME stored
-from previous requests using this ANON_CLIENT_ID.
-
-If either the ORIGIN_NAME or ANON_ORIGIN_ID_PRIME values do not match, the Issuer MUST return an
-HTTP 400 error to the Mediator, which will forward the error to the client.
-
-If the values do match, the Issuer then can use its stored history of activity and token issuance
-to determine if the Client is allowed to receive a token for this Origin during the current policy
-window. If the Issuer refuses to issue more tokens, it responds with an HTTP 429 (Too Many Requests)
-error to the Mediator, which will forward the error to the client.
+If the request is valid, the Issuer then can use the value from "Client-Token-Count" to determine if
+the Client is allowed to receive a token for this Origin during the current policy window. If the
+Issuer refuses to issue more tokens, it responds with an HTTP 429 (Too Many Requests) error to the
+Mediator, which will forward the error to the client.
 
 The Issuer determines the correct ORIGIN_TOKEN_KEY by using the decrypted ORIGIN_NAME value. Issuers
 are expected to be able to deterministically select the correct key based on information sent in
-the request. Clients do not indicate the ORIGIN_TOKEN_KEY to use, to prevent Origins from choosing per-client keys.
+the request. Clients do not indicate the ORIGIN_TOKEN_KEY to use, to prevent Origins from choosing
```

Comment:
Does this get leaked by the choice of key that the ENCRYPTED_ORIGIN_NAME was encrypted to? (can't the Origin provide per-client ISSUER_NAME_KEYs?)

Reply:
The Mediator is responsible for checking the key ID of ISSUER_NAME_KEY to prevent this.